rekurencja/avdiienko.danylo2

```diff
@@ -0,0 +1,13 @@
+x=1
+y=0
+lista=[]
+while x < 100000000:
```

Comment:
a tak w ogóle na przyszlosc to... fajnie jest w kodzie tlumaczyc co jaki element robi:) tzn tak po prostu - komentarzami:)